src/widgets/icons.js

```diff
@@ -86,3 +86,13 @@ export const ChevronRightIcon = () => (
 );
 
 export const FavouriteStarIcon = () => <FAIcon name="star-o" color={SUSSOL_ORANGE} size={20} />;
+export const BurgerMenuIcon = () => <EntypoIcon name="menu" color={SUSSOL_ORANGE} size={30} />;
```

Comment:
three different icon libraries?! does that mean we have an inconsistent look or something?

Reply:
I've made this `icons` file to try and get it more consistent.. previously we had 3 different magnifying glass icons! But yes.. and I think it's 6 icon libraries.... :)